docs/ako_evh.md

```diff
@@ -1,50 +1,48 @@
 # Enhanced Virtual Hosting support in AKO
 
-This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. Currently, this feature is supported only for Kubernetes Ingress object.
+This feature supports EVH VS creation from the AKO. `AKOSettings.enableEVH` needs to be set to `true` to enable this feature. This feature is supported for Kubernetes Ingress object and Openshift Route object.
```

Comment:
@sudswasavi @monotosh-avi : should we specify here what different Openshift deployments we support in EVH explicitly?

Reply:
Do you mean with different CNIs for openshift or the version ?

Reply:
@monotosh-avi : Different deployment modes like secure, insecure , edgetermination(allow/none/redirect) etc

Reply:
I don't think we have to mention those, Ingress also has secure and insecure modes.